apache/couchdb-fauxton
# Add Search support

## Overview
Adds a search module to Fauxton. The functionality is only enabled
upon detection of "search" in the reported CouchDB features.

When enabled, it adds:
 * New dropdown options to create/update search indexes in the sidebar
 * New panel to run search queries from the sidebar
 * Text index templates to the Mango Index editor

## Testing recommendations

Using CouchDB with clouseau (e.g. using the Helm chart), verify that you can:
 * Create a new search index
 * Update a search index
 * Query a search index
 * Create a new Mango text index

From the UI.

For CouchDB 2 / master without clouseau deployed, there should be no options for search indexes in the UI and no errors.

## GitHub issue number



## Related Pull Requests



## Checklist
